src/components/Carousel.tsx

```diff
@@ -1,18 +1,209 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
 import React from 'react';
 import './Carousel.scss';
+import './variables.scss';
+import ClassNames from 'classnames';
 
-const Carousel: React.FC = () => (
-  <div className="Carousel">
-    <ul className="Carousel__list">
-      <li><img src="./img/1.png" alt="1" /></li>
-      <li><img src="./img/1.png" alt="2" /></li>
-      <li><img src="./img/1.png" alt="3" /></li>
-      <li><img src="./img/1.png" alt="4" /></li>
-    </ul>
-
-    <button type="button">Prev</button>
-    <button type="button">Next</button>
-  </div>
-);
+const imagesSrc: string[] = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
+class Carousel extends React.Component<Props, State> {
+  state = {
+    counter: 0,
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+  };
+
+  render() {
+    let { frameSize } = this.state;
+    const {
+      counter,
+      step,
+      itemWidth,
+      animationDuration,
+    } = this.state;
+
+    frameSize *= itemWidth;
+
+    return (
+      <div className="Carousel">
+        <div className="Carousel-container">
+          <ul
+            className="Carousel__list"
+            style={
+              {
+                width: `${frameSize}px`,
+                overflow: 'hidden',
```

Comment:
```suggestion
                overflow: 'hidden',
```
Чому б це не вписати в клас Carousel__list?